diff --git a/processes.json b/processes.json
index c47cb4f..a40dfab 100644
--- a/processes.json
+++ b/processes.json
@@ -4,10 +4,10 @@
       "name": "mirakurun-server",
       "script": "lib/server.js",
       "node_args" : "--max-semi-space-size=64 -r source-map-support/register",
-      "error_file": "/usr/local/var/log/mirakurun.stderr.log",
-      "out_file": "/usr/local/var/log/mirakurun.stdout.log",
+      "error_file": "/var/log/mirakurun.stderr.log",
+      "out_file": "/var/log/mirakurun.stdout.log",
       "merge_logs": true,
-      "pid_file": "/usr/local/var/run/mirakurun.pid",
+      "pid_file": "/var/run/mirakurun.pid",
       "exec_mode": "fork",
       "autorestart": true,
       "env": {
diff --git a/src/Mirakurun/config.ts b/src/Mirakurun/config.ts
index c36e9ca..b97e050 100644
--- a/src/Mirakurun/config.ts
+++ b/src/Mirakurun/config.ts
@@ -17,7 +17,7 @@
 import { dirname } from "path";
 import { hostname } from "os";
 import { existsSync, readdirSync } from "fs";
-import { mkdir, copyFile, readFile, writeFile } from "fs/promises";
+import { mkdir, copyFile, chmod, readFile, writeFile } from "fs/promises";
 import * as yaml from "js-yaml";
 import * as ipnum from "ip-num";
 import Queue from "promise-queue";
@@ -83,6 +83,7 @@
         try {
             log.info("copying default server config to `%s`", path);
             await copyFile("config/server.yml", path);
+            await chmod(path, 0o644);
         } catch (e) {
             log.fatal("failed to copy server config to `%s`", path);
             console.error(e);
@@ -328,6 +329,7 @@
         try {
             log.info("copying default tuners config to `%s`", path);
             await copyFile("config/tuners.yml", path);
+            await chmod(path, 0o644);
         } catch (e) {
             log.fatal("failed to copy tuners config to `%s`", path);
             console.error(e);
@@ -365,6 +367,7 @@
         try {
             log.info("copying default channels config to `%s`", path);
             await copyFile("config/channels.yml", path);
+            await chmod(path, 0o644);
         } catch (e) {
             log.fatal("failed to copy channels config to `%s`", path);
             console.error(e);
